pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -62,17 +62,17 @@ self: super: {
 
   # The Hackage tarball is purposefully broken, because it's not intended to be, like, useful.
   # https://git-annex.branchable.com/bugs/bash_completion_file_is_missing_in_the_6.20160527_tarball_on_hackage/
-  git-annex = ((overrideCabal super.git-annex (drv: {
+  git-annex = dontCheck (((overrideCabal super.git-annex (drv: {
```

Comment:
Please do not disable the tests. `git-annex` is used for manipulating important data and I don't want to distribute a version of the tool that does not pass it's own test suite.